client/landing/gutenboarding/components/domain-picker-button/style.scss

```diff
@@ -1,24 +1,44 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '../../mixins.scss';
 
-@keyframes gutenboarding_wriggle {
+@keyframes domain-picker-button-slide-in {
 	from {
-		transform: scale( 1 );
-		padding: 0;
+		max-width: 0;
+		opacity: 0;
+		padding: 5px 0;
 	}
 
-	50% {
-		transform: scale( 1.05 );
-		padding: 2px;
+	20% {
+		max-width: 0;
+		opacity: 0;
+		padding: 5px 12px;
 	}
 
 	to {
-		transform: scale( 1 );
-		padding: 0;
+		max-width: 500px;
+		opacity: 1;
+		padding: 5px 12px;
 	}
 }
 
 .domain-picker-button {
+	&--has-first-content {
+		animation-duration: 1.6s;
+		animation-name: domain-picker-button-slide-in;
+		max-width: 1500px;
+		@include reduce-motion( 'animation' );
+	}
+
+	&--has-placeholder {
+		@include onboarding-placeholder;
+		@include reduce-motion( 'animation' );
```

Comment:
Shouldn't the `onboarding-placeholder` just directly include `reduce-motion`? Not a biggie for this PR.